versioned_docs/version-8.6/reference/supported-environments.md

```diff
@@ -119,7 +119,9 @@ Camunda 8 works with the [default distribution](https://www.elastic.co/downloads
 
 ### Component version matrix
 
-This matrix shows which component versions work together:
+This matrix shows which component versions work together.
+
+From version `8.6.0` forward, Zeebe, Operate, and Tasklist must run on on the exact same `minor` and `patch` level to ensure compatibility.
```

Comment:
I added the note here as well, but did not update the table to match the version used in 8.7. If we decide to update this, I can modify it in a separate PR. 👍